nncf/tensorflow/helpers/model_creation.py

```diff
@@ -74,8 +73,6 @@ def create_compressed_model(model: tf.keras.Model,
             original_model_accuracy = evaluation_args.eval_fn(model)
 
     builder = create_compression_algorithm_builder(config, should_init)
-    if builder is None:
-        return None, model
```

Comment:
@andrey-churkin Please fix also the docstring and type hint for return parameters.

Reply:
Done